content/guides/authentication/sso.md

```diff
@@ -38,11 +37,12 @@ Box account.
 ## Custom Apps with Server-side Authentication
 
 In any other situation a Custom App is authenticated using server-side
-technology and no actual Managed User is involved. For this reason, SSO is not
-involved in this process. These applications often
+technology and no actual Managed User is involved. These applications often  
+utilize their own user-facing authentication flows.
```

Comment:
I'm not too sure about this text. I don't want to get too much in what could-be's. Let's stick to the facts. A server-side authenticated app might not use SSO as all as there's might not be any direct user-to-Box-user mapping.

Reply:
I'd like to make it very clear that at this point, user auth is the app's prerogative, not ours.

Reply:
Agreed, re-organized this section to try and be more clear and emphasize that end user auth is determined by the app, and not Box.